eos-updater-flatpak-installer/docs/eos-updater-flatpak-installer.8

```diff
@@ -0,0 +1,92 @@
+.\" Manpage for eos-updater-flatpak-installer.
+.\" Documentation is under the same licence as the eos-updater package.
+.TH man 8 "13 Apr 2017" "1.0" "eos\-updater\-flatpak\-installer man page"
```

Comment:
Note to self: I said I’d look at tidying up this man page (if needed) so the markup is consistent with other man pages and it follows all the troff/groff rules.

Reply:
Hmm - actually that was the one thing I was trying to test the other day was getting the man pages to actually *install* correctly. I mostly cargo-culted the avahi manpage in this case (but I'm pretty ignorant of how man pages actually work - this is the first one that I've written).

Reply:
Installing them should be a matter of putting them in the right directory (via `Makefile.am` and `debian/*.install`). They don’t need any post-processing on installation.